### PR TITLE
fix: Translation and Grammar fixes for Chinese locPak (Final)

### LINKED ISSUE #4188: zh-CN locpack Grammar and Translation Issues
**Mod Version**
All Versions

**Describe the bug**
My Taiwanese buddies has been using this mod,  found some mistranslations in the locpak of zh-CN

**To Reproduce**
 N/A

**Expected behavior**
Correctly translated characters for the native people to understand.

**Actual behavior**
Some mistakes, as shown below in the references.

**References**
transissu

Discord username (if different from GitHub): chiar.hx#3432

### PR DESCRIPTION
**Pull request by Asianblade**
Fixes #4188

## Summary of Changes
Changed various translation and grammatical mistakes for the Chinese locPak.

## Screenshots (if necessary)
N/A (Shown in code)

## References
N/A

## Additional context
N/A

Discord username (if different from GitHub): Wesley#0008 (Wesley#TripleZero8)

## Testing instructions
Open the locPak, and scan through it.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
2. Click on the **Checks** tab on the PR
3. On the left side, click on the bottom **PR** tab
4. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Comment by Saschl** [comment]
Please don't open multiple PRs. You can edit existing ones. 

And change your profile picture. That's not acceptable.

**Comment by Asianblade** [comment]
Reply to Saschl:
1. Profile picture changed
2. Can't revert wrong checks, (or I don't know how to), this time they merged and sorry about that

File: A32NX/zh-CN.locPak [hunk 1]
```diff
@@ -2,19 +2,19 @@
...
       "COCKPIT.TOOLTIPS.AUTOPILOT_PANEL_HDG_TUNING_RIGHT": "向右调整所选航向(%((A:AUTOPILOT HEADING LOCK DIR:1,degrees))%!d!°)",
       "COCKPIT.TOOLTIPS.APU_AVAILABLE": "APU available",
       "A32NX.LOADING.TIPS.A320NEO_001": "FlyByWire A32NX是由数百名个人贡献者合作完成的。",
-      "A32NX.LOADING.TIPS.A320NEO_002": "你知道吗？A320的横风速度可达38节，尾风速度可达15节！A320可以以38节的速度降落。",
+      "A32NX.LOADING.TIPS.A320NEO_002": "你知道吗？A320的降落横风速度可达38节，尾风速度可达15节!",
       "A32NX.LOADING.TIPS.A320NEO_003": "FlyByWire的名字来源于空客A320首次引入的飞线控制。",
       "A32NX.LOADING.TIPS.A320NEO_004": "A320家族飞机共生产9693架! 这几乎和图瓦卢的人口一样多!",
       "A32NX.LOADING.TIPS.A320NEO_005": "空客A320neo飞机的航程为3500NM或6500KM。这足以从纽约飞到巴黎!",
-      "A32NX.LOADING.TIPS.A320NEO_006": "你知道吗？A320家族有4个变种! A318 A319 A320和A321。"
+      "A32NX.LOADING.TIPS.A320NEO_006": "你知道吗？A320家族有4种类型! A318 A319 A320和A321。"
```
Comment (cmp831) [inline A32NX/zh-CN.locPak:17]: I think "型号" will be better suited when referencing "variants" instead of "类型"

File: A32NX/zh-CN.locPak [hunk 2]
```diff
@@ -2,19 +2,19 @@
...
+      "AIRCRAFT.DESCRIPTION": "空中客车公司（Airbus）于1984年3月推出了第一架A320。近26年后的2010年，他们宣布A320neo（“新发动机选件”）成为家族的最新成员，以振兴和维持其世界领先的产品线，在经过扩展的开发过程之后，A320neo于2016年初在汉莎航空正式服役，并迅速成为有史以来销售最快的商用飞机，迄今为止，已有115多家航空公司订购了7,000架飞机。延续了最初的A320建立的传统，A320是除协和飞机以外的第一架采用电传操纵进行正常运行的客机， A320neo与之前的机型共享95％以上的机身通用性。但是，正是引入了两种新一代发动机类型，再加上该公司专有的“ Sharklet”翼尖，才真正使Neo脱颖而出。有效载荷和缩短的着陆距离；这些重要的进步-以及噪音和CO 2排放量的大幅度减少-证明了A320neo不仅在商业上取得成功，而且在航空业对环境承担更大责任的道路上迈出了重要的一步。",
       "AIRCRAFT.UI_MANUFACTURER": "Airbus",
       "AIRCRAFT.UI_MODEL": "A320neo",
       "AIRCRAFT.UI_VARIATION": "Airbus",
       "COCKPIT.TOOLTIPS.AUTOPILOT_PANEL_HDG_TUNING_LEFT": "向左调整所选航向(%((A:AUTOPILOT HEADING LOCK DIR:1,degrees))%!d!°)",
       "COCKPIT.TOOLTIPS.AUTOPILOT_PANEL_HDG_TUNING_RIGHT": "向右调整所选航向(%((A:AUTOPILOT HEADING LOCK DIR:1,degrees))%!d!°)",
       "COCKPIT.TOOLTIPS.APU_AVAILABLE": "APU available",
       "A32NX.LOADING.TIPS.A320NEO_001": "FlyByWire A32NX是由数百名个人贡献者合作完成的。",
-      "A32NX.LOADING.TIPS.A320NEO_002": "你知道吗？A320的横风速度可达38节，尾风速度可达15节！A320可以以38节的速度降落。",
+      "A32NX.LOADING.TIPS.A320NEO_002": "你知道吗？A320的降落横风速度可达38节，尾风速度可达15节!",
```
Comment (cmp831) [inline A32NX/zh-CN.locPak:13]: For aviation, the term "侧风" seems more popular than "横风"

File: A32NX/zh-CN.locPak (hunk 1 above)
Comment (Asianblade) [reply A32NX/zh-CN.locPak:17]: Passed

**Review by Benjozork — APPROVED** [review]
LGTM

**Comment by Benjozork** [comment]
Please rebase on latest master by applying changes to locPak file which moved directories.

**Comment by Asianblade** [comment]
Resolved.

**Pull request merged**